src/schneider_lab_to_nwb/schneider_2024/schneider_2024_convert_session.py

```diff
@@ -2,68 +2,93 @@
 from pathlib import Path
 from typing import Union
 import datetime
-from zoneinfo import ZoneInfo
+import pytz
+import shutil
+from pprint import pprint
+import numpy as np
 
 from neuroconv.utils import load_dict_from_file, dict_deep_update
-
 from schneider_lab_to_nwb.schneider_2024 import Schneider2024NWBConverter
 
 
 def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str, Path], stub_test: bool = False):
 
     data_dir_path = Path(data_dir_path)
     output_dir_path = Path(output_dir_path)
+    recording_folder_path = data_dir_path / "Raw Ephys" / "m69_2023-10-31_17-24-15_Day1_A1"
+    sorting_folder_path = data_dir_path / "Processed Ephys" / "m69_2023-10-31_17-24-15_Day1_A1"
     if stub_test:
         output_dir_path = output_dir_path / "nwb_stub"
+        recording_folder_path = recording_folder_path.with_name(recording_folder_path.name + "_stubbed")
     output_dir_path.mkdir(parents=True, exist_ok=True)
 
-    session_id = "subject_identifier_usually"
+    session_id = "sample_session"
     nwbfile_path = output_dir_path / f"{session_id}.nwb"
 
     source_data = dict()
     conversion_options = dict()
 
     # Add Recording
-    source_data.update(dict(Recording=dict()))
+    stream_name = "Signals CH"  # stream_names = ["Signals CH", "Signals AUX"]
+    source_data.update(dict(Recording=dict(folder_path=recording_folder_path, stream_name=stream_name)))
     conversion_options.update(dict(Recording=dict(stub_test=stub_test)))
 
     # Add Sorting
-    source_data.update(dict(Sorting=dict()))
+    source_data.update(dict(Sorting=dict(folder_path=sorting_folder_path)))
     conversion_options.update(dict(Sorting=dict()))
 
-    # Add Behavior
-    source_data.update(dict(Behavior=dict()))
-    conversion_options.update(dict(Behavior=dict()))
+    # # Add Behavior
+    # source_data.update(dict(Behavior=dict()))
+    # conversion_options.update(dict(Behavior=dict()))
 
     converter = Schneider2024NWBConverter(source_data=source_data)
 
     # Add datetime to conversion
     metadata = converter.get_metadata()
-    datetime.datetime(
-        year=2020, month=1, day=1, tzinfo=ZoneInfo("US/Eastern")
-    )
-    date = datetime.datetime.today()  # TO-DO: Get this from author
-    metadata["NWBFile"]["session_start_time"] = date
+    EST = pytz.timezone("US/Eastern")
+    metadata["NWBFile"]["session_start_time"] = EST.localize(metadata["NWBFile"]["session_start_time"])
 
     # Update default metadata with the editable in the corresponding yaml file
     editable_metadata_path = Path(__file__).parent / "schneider_2024_metadata.yaml"
     editable_metadata = load_dict_from_file(editable_metadata_path)
     metadata = dict_deep_update(metadata, editable_metadata)
 
     metadata["Subject"]["subject_id"] = "a_subject_id"  # Modify here or in the yaml file
+    conversion_options["Sorting"]["units_description"] = metadata["Ecephys"]["UnitProperties"][0]["description"]
```

Comment:
Is this the description of the units table?

Reply:
yes

Reply:
It seems wrong to me that the place that is in the metadata. Probably not important but I feel this maybe should go somewhere else. Isn't that list for properties of the columns?

Reply:
> Isn't that list for properties of the columns?

Yes.

It probably should go somewhere else, but I can't add another field to the Ecephys metadata without the schema yelling at me. I prefer to have all the metadata I can in the .yaml file so that it's all collected in one place, but the SortingExtractor has this one description in the conversion_options for some reason?

Anyway I realize using UnitProperties in this way is a bit hacky, so I will move it outside of Ecephys altogether.